.gitattributes

```diff
@@ -2,3 +2,6 @@
 # Set default behavior to automatically normalize line endings.
 ###############################################################################
 * text=auto
+
+*.vue text eol=lf
```

Comment:
not sure this is what we want since I think all files should be in lf. Thoughts?

Reply:
want me to change it to `* text eol=lf`?

Reply:
I think we do!

Reply:
Uhm not sure. Do we also want the CS and json files to be lf?